### ISSUE TITLE
Touchscreen usability: hover text or status info

### ISSUE THREAD

**Issue by j5v** [j5v/xetune#9]
Mouseover `title` text, while helpful to mouse users, cannot be viewed on a touch device.

As a mobile user, I want to see more precise details for analysis cells, where details are only displayed as mouseover text.

**Comment by j5v** [comment]
1. I want to see full information for a note.

This might be fixed by the upcoming feature **Note properties** #11

**Comment by j5v** [comment]
Removed the old point 1, mentioned in previous comment.
